src/liballoc/tests/btree/map.rs

```diff
@@ -25,6 +26,20 @@ const MIN_INSERTS_HEIGHT_1: usize = NODE_CAPACITY + 1;
 // It's not the minimum size: removing an element from such a tree does not always reduce height.
 const MIN_INSERTS_HEIGHT_2: usize = NODE_CAPACITY + (NODE_CAPACITY + 1) * NODE_CAPACITY + 1;
 
+// Gather all references from a mutable iterator and make sure Miri notices if
+// using them is dangerous.
+fn test_all_refs<'a, T: 'a>(dummy: &mut T, iter: impl Iterator<Item = &'a mut T>) {
```

Comment:
I added a `cfg_attr(miri, ignore)` to the min_max test, could you also remove that?

Reply:
During the next rebase... at which point, do I also squash your commit or keep that separate?

Reply:
IMO it makes sense to keep it separate, but I have no strong opinion either way.